fxa-auth-db-server/index.js

```diff
@@ -135,6 +135,9 @@ function createServer(db) {
   api.post('/verificationReminders', replyNoParams(db.createVerificationReminder))
   api.del('/verificationReminders', replyNoParams(db.deleteReminder))
 
+  api.get('/securityEvents', reply(db.securityEvents))
+  api.post('/securityEvents', reply(db.securityEvent))
```

Comment:
Actually, one more thing: could we differentiate the names of these two methods a little bit? For the other write methods we use `createXxxx`.